FTP Front-end Project/src/components/GameCard/GameCard.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+const GameCard = ({ game }) => {
+  return (
+    <li className="game">
+      <img className="game__img" src={game.thumbnail} />
```

Comment:
Every `img` tag must have an **descriptive** `alt` attribute. The `alt` attribute provides **alternative information** for an image if a user, for some reason, cannot view it (because of slow connection, an error in the `src` attribute, or if the user is using a screen reader). Well-written `alt` text is important to your website's accessibility, and its search engine optimization. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img https://moz.com/learn/seo/alt-text

**Please, correct it everywhere in the code**